### PR TITLE
router-perf job fails when benchmark comparison fails

### PR DESCRIPTION
**Pull request by qiliRedHat**
Depends on PR https://github.com/cloud-bulldozer/e2e-benchmarking/pull/535 to merge
Add COMPARISON_RC (default=1) to let Benchmark-comparison return 1 if tolerancy check fails.
Add Tolerancy to replace tolerancy in [mb-tolerancy-rules.yaml](https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/router-perf-v2/mb-tolerancy-rules.yaml)

### CODE REVIEW

**Comment by qiliRedHat** [comment]
**Passed job**
TOLERANCY=-1 (if deviation <-1%, touchstone compare fails)
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/qili-e2e-benchmark/job/router-perf/147/
```
02-20 14:30:57.732  [1mMon Feb 20 06:30:56 UTC 2023 Removing touchstone[0m
02-20 14:30:57.732  + grep 'Benchmark comparison failed' ingress_router.out
02-20 14:30:57.737  [Pipeline] sh
```

test_type | routes | conn_per_targetroute | keepalive | metric | result | deviation | 088a73b0-15bc-4a25-993e-1efc32cd87e3 | 5c89c826-2e8a-47a5-a310-d61175c54236
-- | -- | -- | -- | -- | -- | -- | -- | --
mix | 40 | 1 | 0 | avg(requests_per_second) | Pass | 27.84% | 15.254 | 19.501
mix | 40 | 125 | 0 | avg(requests_per_second) | Pass | 61.01% | 20.893 | 33.639



**Failed job** (expected to be failed)
TOLERANCY=-10 (default) (if deviation <-10%, touchstone compare fails)
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/qili-e2e-benchmark/job/router-perf/139/
```
02-20 12:13:15.519  [1mMon Feb 20 04:13:15 UTC 2023 Benchmark comparison failed[0m
```

test_type | routes | conn_per_targetroute | keepalive | metric | result | deviation | 088a73b0-15bc-4a25-993e-1efc32cd87e3 | ebd54d04-7125-4203-99f6-9218aee1aa49
-- | -- | -- | -- | -- | -- | -- | -- | --
mix | 40 | 1 | 0 | avg(requests_per_second) | Pass | -4.45% | 15.254 | 14.575
mix | 40 | 125 | 0 | avg(requests_per_second) | Fail | -16.02% | 20.893 | 17.546

**Comment by qiliRedHat** [comment]
@skordas It's ready to review this now.

File: Jenkinsfile [hunk 1]
```diff
@@ -161,6 +161,22 @@ pipeline {
...
+            defaultValue: '1',
+            description: '''
+                Benchmark-comparison return code if tolerancy check fails
+            '''
+        )
+        string(
+            name: 'TOLERANCY',
+            defaultValue: '-10',
+            description: '''
+                Tolerancy to replace tolerancy in mb-tolerancy-rules.yaml <a href="https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/router-perf-v2/mb-tolerancy-rules.yaml"></a><br>
```
Comment (skordas) [inline Jenkinsfile:175]: Please move mb-tolerancy-rules.yaml into 'a' tag (right now is not displayed as a link)

Comment (qiliRedHat) [reply Jenkinsfile]: @skordas Updated. Thanks!

**Pull request merged**